var/spack/packages/flux/package.py

```diff
@@ -0,0 +1,36 @@
+from spack import *
+import os
+
+class Flux(Package):
+    """ A next-generation resource manager (pre-alpha) """
+
+    homepage = "https://github.com/flux-framework/flux-core"
+    url      = "https://github.com/flux-framework/flux-core"
+
+    version('master', branch='master', git='https://github.com/flux-framework/flux-core')
+
+    # Also needs autotools, but should use the system version if available
+    depends_on("zeromq@4.0.4:")
+    depends_on("czmq@2.2:")
+    depends_on("lua@5.1:5.1.99")
```

Comment:
@trws: did @5.1 not work?  "@5.1" should match 5.1.99, but there may be a bug in the way this is handled in the concretizer.